client/build.js

```diff
@@ -0,0 +1,73 @@
+const { build, cliopts, basename, stdoutStyle: styl, fmtDuration } = require('estrella')
+const { nodeExternalsPlugin } = require('esbuild-node-externals')
+const { readdirSync, realpathSync } = require('fs')
+const { performance } = require('perf_hooks')
+
+const dev = !!cliopts.watch
```

Comment:
I prefer `Boolean(..)` over `!!..` as more readable.

Reply:
This is governed by an ESLint rule, actually. We can make a change for that.